### DISCUSSION TITLE
Missing rules

### DISCUSSION THREAD

**Discussion by robov** [ash-jc-allen/laravel-config-validator#53] [General]
Is it possible to thow an exception on missing validation rules. This ensures that all keys have some validations set and that not keys have been added accidentaly that have no rules set

**Comment by ash-jc-allen** [comment]
Hey @robov! At the moment, there isn't any functionality that checks that every config value has validation. It is something that I did consider, but then I didn't go through with adding. My main reason was that it would require you to write a validation rule for everything single config value that's registered in your app (even if you aren't using them).

But, if this is something that enough people want, I'd consider taking a look at a few different solutions for this 🙂
